packages/suite/src/components/wallet/WalletLayout/components/WalletLayoutNavigation.tsx

```diff
@@ -33,22 +33,45 @@ const NavLink = styled.div<{ active?: boolean }>`
 
 const NavLinkText = styled.div`
     position: relative;
+    z-index: 1;
+`;
+
+const Badge = styled.span`
+    position: absolute;
+    top: -14px;
+    right: -27px;
+    padding: 2px 4px 0 4px;
+    border-radius: 4px;
+    background: ${props => props.theme.BG_LIGHT_GREEN};
+    color: ${props => props.theme.TYPE_GREEN};
+    font-size: 12px;
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    text-transform: uppercase;
+    letter-spacing: 0.2px;
+    cursor: default;
```

Comment:
since you change the cursor, I would add:

```suggestion
    pointer-events: none;
```

Reply:
I saw this change of cursor in other places. Not sure it's necessary. Anyway, I added the suggested property.

Reply:
It's a tiny detail, but IMO these properties should be in line. So I think those other places have it wrong, feel free to fix it when you come over it again.